usr/lib/cca_stdll/cca_specific.c

```diff
@@ -4409,6 +5250,11 @@ CK_RV token_specific_generic_secret_key_gen(STDLL_TokData_t * tokdata,
         return rc;
     }
 
+#ifdef DEBUG
+    TRACE_DEBUG("%s: imported object template attributes:\n", __func__);
+    dump_template(template);
```

Comment:
If you add this here (which makes sense, now that we have this functions), why not also adding it to the other keygen functions as well?
Any maybe adjust the trace message, here is is not an 'imported' key, but a 'generated' one.